pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -183,7 +186,12 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, pr *v1beta1.PipelineRun)
 		before = pr.Status.GetCondition(apis.ConditionSucceeded)
 	}
 
-	getPipelineFunc := resources.GetPipelineFunc(ctx, c.KubeClientSet, c.PipelineClientSet, c.resolutionRequester, pr)
+	// get verificationpolicies for trusted resources
+	vp, err := getVerificationPolicies(ctx, c.verificationPolicyLister, pr.Namespace)
+	if err != nil {
+		return err
```

Comment:
please add more context onto the error (also a general piece of feedback whenever an error is being returned)